react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts

```diff
@@ -46,18 +67,15 @@ function mapToRange(resolved_mesh: MeshType, meshValueRange: [number, number]) {
 function add_normals(
     resolved_mesh: MeshType,
     meshImageData: ImageData,
-    bounds: [number, number, number, number]
+    bounds: BoundsType
```

Comment:
I think we need a new name for this prop, so that we can keep but deprecate the old prop for a while.

Also, I think we could be more explicit in the `BoundsType`. For example:

```
bounds: {
  increment: [number, number],
  origin: [number, number],
  count: [number, number],
}

We can also make `count` optional in this case, because it's given by the data structure, but could serve as an assertion.

Reply:
Maybe I can rename bonods to horizontalExtent. I think I need count otherwise it will not be possible to caclutale xmin, xmax ..

Reply:
> Maybe I can rename bonods to horizontalExtent. I think I need count otherwise it will not be possible to caclutale xmin, xmax ..

You have `ni` and `nj` given by the png image format. You can calculate the x and y extents from `xIncrement * ni` and `yIncrement * nj`

Reply:
> Maybe I can rename bonods to horizontalExtent.

How about "frame"?